core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -543,40 +544,22 @@ public int normalizeLimit(int suppliedLimit) {
   }
 
   @Override
-  public int register(String subject,
-                      Schema schema,
-                      boolean normalize)
+  public Schema register(String subject,
```

Comment:
Would be useful if we can have a summary of the register() function.

Reply:
I added JavaDoc